### PR TITLE
[GOLD] Adapting scripts for new TBB

### PR DESCRIPTION
**Pull request by homksei**

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -207,15 +207,15 @@ frompf1 = $(shell echo $1 | sed 's/111/\\ /g' | sed 's/222/(/g' | sed 's/333/)/g
 #============================= TBB folders =====================================
 TBBDIR := $(if $(wildcard $(DIR)/__deps/tbb/$(_OS)/*),$(DIR)/__deps/tbb/$(_OS)$(if $(OS_is_win),/tbb))
 TBBDIR.2 := $(if $(TBBDIR),$(TBBDIR),$(call topf,$$TBBROOT))
-TBBDIR.2 := $(if $(TBBDIR.2),$(TBBDIR.2),$(error Can`t find TBB neither in $(DIR)/__deps/tbb nor in $$TBBROOT))
+TBBDIR.2 := $(if $(TBBDIR.2),$(TBBDIR.2),$(error Can`t find TBB neither in $(DIR)/__deps/tbb not in $$TBBROOT))
```
Comment (averbukh) [inline makefile:210]: Before your changes it was correct: "neither... nor..."

**Review by averbukh — CHANGES_REQUESTED** [review]

**Review by averbukh — APPROVED** [review]

**Comment by homksei** [comment]
/intelci: run
set infra_branch=dev/alekseik/GOLD/Tools_update

**Comment by napetrov** [comment]
@Mergifyio backport master

**Pull request merged**